defaults/main/template.yml

```
@@ -278,6 +281,16 @@ nginx_config_http_template:
         max_concurrent_streams: 128  # Number -- Not available in the 'location' context
         recv_buffer_size: 256k  # Only available in the 'http' context
         recv_timeout: 20s  # Not available in the 'location' context
+      http3:
+        enable: false # Boolean
```

```suggestion
        enable: false  # Boolean
```
Let's add two spaces between "directives" and "comments". Helps with readability :)